scripts/scp_to_manifest.py

```diff
@@ -17,13 +17,17 @@
 import os
 
 import librosa as l
+from examples.speaker_recognition.speaker_reco import logging
 from sklearn.model_selection import StratifiedShuffleSplit
 from tqdm import tqdm
 
+import nemo
+
+logging = nemo.logging
 
 """
-This scipt take scp file which is in format 
-<absolute  filename> 
+This scipt take scp file where each line contains  
```

Comment:
Needs to be either `converts scp files ... to manifest json files` or `converts a scp file ... to a manifest json file`

Reply:
sure, noted